src/Views/Filter/Subject.cshtml

```diff
@@ -1,33 +1,60 @@
 @model SubjectFilterViewModel
 
 @{
-    ViewBag.Title = $"Filter by subject";
-    var selected = Model.FilterModel.SelectedSubjects;
-    var isInWizard = ViewBag.IsInWizard == true;
-    var errors = ViewBag.Errors as ErrorViewModel;
+  var selected = Model.FilterModel.SelectedSubjects;
+  var isInWizard = ViewBag.IsInWizard == true;
+  var errors = ViewBag.Errors as ErrorViewModel;
 }
 
 
 @if(isInWizard) {
-    @Html.Partial("Back", new BackLinkViewModel {
-        Href = Url.Action("LocationWizard", "Filter", Model.FilterModel.ToRouteValues()),
-        Title = "Back to location"
-    })
+  @Html.Partial("Back", new BackLinkViewModel {
+    Href = Url.Action("LocationWizard", "Filter", Model.FilterModel.ToRouteValues()),
+    Title = "Back to location"
+  })
 } else {
-    @Html.Partial("Back", new BackLinkViewModel {
-        Href = Url.Action("Index", "Results", Model.FilterModel.ToRouteValues()),
-        Title = "Back to search results"
-    })
+  @Html.Partial("Back", new BackLinkViewModel {
+    Href = Url.Action("Index", "Results", Model.FilterModel.ToRouteValues()),
+    Title = "Back to search results"
+  })
 }
 
 <main class="govuk-main-wrapper" id="main-content" role="main">
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">Find courses by subject</h1>
-            <p class="govuk-body">
-            <p class="govuk-body">
-                <a href='@Url.Action("Index", "Results", Model.FilterModel.ToRouteValues())' class="govuk-link">Back to results</a>
-            </p>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      @Html.Partial("Error", errors)
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset" aria-describedby="nationality-hint">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+            <h1 class="govuk-fieldset__heading">What subjects would you like to&nbsp;teach?</h1>
```

Comment:
Can the title be "Find courses by subject" (see https://search-and-compare-prototype.herokuapp.com/start/subjects)